checkov/common/output/common.py

```diff
@@ -15,3 +15,32 @@ class ImageDetails(SCADetails):
     image_id: str = ''
     name: str | None = ''
     related_resource_id: str | None = ''
+
+
+def is_raw_formatted(licenses: str) -> bool:
+    return licenses.__contains__('","')
+
+
+def format_licenses_to_string(licenses_lst: list[str]) -> str:
+    print('format_licenses_to_string - start', licenses_lst)
+    if licenses_lst and len(licenses_lst) > 1:
+        joined_str = '","'.join(licenses_lst)
+        return f'"{joined_str}"'
+    elif licenses_lst and len(licenses_lst) == 1:
+        return licenses_lst[0]
+    else:
+        return 'Unknown'
```

Comment:
```suggestion
    if isinstance(licenses_lst, list):
        if len(licenses_lst) > 1:
            joined_str = '","'.join(licenses_lst)
            return f'"{joined_str}"'
        elif licenses_lst:
            return licenses_lst[0]
    return 'Unknown'
```
depends on what you want to do and how sure you are that a list is passed to the function.
```suggestion
    if len(licenses_lst) > 1:
        joined_str = '","'.join(licenses_lst)
        return f'"{joined_str}"'
    elif licenses_lst:
        return licenses_lst[0]
    else:
        return 'Unknown'
```